core/src/test/resources/assets/sql/missing_last_semicolon.sql

```diff
@@ -1,10 +0,0 @@
--- We ommit the trailing semicolon of the last statement to test whether the parser detects that.
-CREATE TABLE 'testTable'
```

Comment:
Moved these from files to Strings as they're small and we had issues getting the tests to read these files